src/exception.rs

```diff
@@ -67,6 +71,18 @@ extern "C" {
 }
 
 impl StackTrace {
+  /// Grab a snapshot of the current JavaScript execution stack.
+  pub fn current_stack_trace<'s>(
+    scope: &mut HandleScope<'s>,
+    frame_limit: i32,
```

Comment:
Consider using `usize` and casting to `i32`. V8 uses `int` extensively because Google and their C++ style guide but that doesn't really carry over to Rust land.

Reply:
Done